api/alembic/versions/c9e46d098c73_critical_hours.py

```diff
@@ -0,0 +1,57 @@
+"""Critical hours
+
+Revision ID: c9e46d098c73
+Revises: 6910d017b626
+Create Date: 2024-08-12 16:24:00.489375
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "c9e46d098c73"
+down_revision = "6910d017b626"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
```suggestion
```

Reply:
Done in: https://github.com/bcgov/wps/pull/3849/commits/af4bd2af7d9680afcf95ab7b6e8537fcb87cdbe9